koku/cost_models/serializers.py

```diff
@@ -395,6 +434,8 @@ class Meta:
         choices=metric_constants.DISTRIBUTION_CHOICES, required=False, allow_blank=True
     )
 
+    distribution_info = DistributionSerializer(required=False)
```

Comment:
why do we have:
```
distribution = serializers.ChoiceField(        
    choices=metric_constants.DISTRIBUTION_CHOICES, required=False, allow_blank=True	        )
```
and 
```
distribute_type = serializers.ChoiceField(choices=metric_constants.DISTRIBUTION_CHOICES, required=False)
```
Aren't these the same thing?

Reply:
After reading the JIRA, I guess we are moving the distribution into the `distribution_info` json as the `distribution_type` field? Is that correct?

Reply:
we are keeping the distribution field for now, to be removed in a followup clean up step